src/app/about/about.component.ts

```diff
@@ -10,5 +10,6 @@ import { Component } from '@angular/core';
     templateUrl: './about.component.html'
 })
 export class AboutComponent {
+    open: Boolean = false;
```

Comment:
Hi @pratheekhegde, thank you for this PR. Could you point out why this `open` property was added in? How is it used?